components/widgets/resources/src/main/resources/resources/uicomponents/widgets/exportData.js

```diff
@@ -188,35 +188,73 @@ document.observe('xwiki:dom:loading', function() {
     }
     
     //============================================================================
-    // Column selection
-    var columnList = content.down('.section.columns ul');
-    if (columnList) {
+    // Push dialog check boxes and expand tools
+    var titles = content.select('.push-fields.section.columns > h5');
+      if (titles) {
+        titles.each(function(item, index){
+        var sectionList = item.next('ul');
+        var showIcon = '<span class="fa fa-plus-square-o fa-lg"></span>';
+        var chapterShow = new Element('button', {'class' : 'tool button secondary', 'type' : 'button'}).update(showIcon+" "+"$services.localization.render('phenotips.patientSheet.expandSection')");
+        var hideIcon = '<span class="fa fa-minus-square-o fa-lg"></span>';
+        var chapterHide = new Element('button', {'class' : 'tool button secondary', 'type' : 'button'}).update(hideIcon+" "+"$services.localization.render('phenotips.patientSheet.collapseSection')");
+        var chapterShowWrapper = new Element('span', {'class' : 'buttonwrapper show'}).insert(chapterShow);
+        var chapterHideWrapper = new Element('span', {'class' : 'buttonwrapper hide'}).insert(chapterHide);
+        var chapterExpandTools = new Element('span', {'class' : 'expand-tools'}).insert(chapterShowWrapper).insert(chapterHideWrapper);
+        item.insert({after: chapterExpandTools});
+        // section expand tools behaviour
+        [chapterShow, chapterHide, item].invoke('observe', 'click', function (event) {
+          if (event.path[0].tagName != "INPUT") {
```

Comment:
event.path is not implemented in firefox. To find to original element, you could use prototype's event.findElement() instead of event.path[0].